app/templates/eligible.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/input/macro.html' import govukInput -%}
+
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+
+{% block content %}
+
+<div>
+  <h2 class="govuk-heading-l">You're eligible for this fund.</h2>
+    <p  class="govuk-body-l">
+      Based on your answers, you are eligible to apply for this fund.
+    </p>
+</div>
+
+{{ govukButton({
+    "isStartButton": True,
+    "href" : service_url,
+  "text": "Start now"
+}) }}
```

Comment:
Where do you find these button options?

Reply:
The components and their options can be found here https://govuk-frontend-jinja.herokuapp.com